privacyidea/static/components/config/views/config.resolvers.httpresolver.html

```diff
@@ -0,0 +1,99 @@
+                    required>
+                    <option value="get">GET</option>
+                    <option value="post">POST</option>
+                </select>
+            </div>
+        </div>
+        <div class="form-group row">
+            <label for="requestmapping" class="col-sm-3 control-label text-right" translate>Request Mapping (JSON format)</label>
+
+            <div class="col-sm-9">
+                <textarea name="requestmapping" id="requestmapping" ng-model="params.requestMapping"
+                    class="form-control" required></textarea>
+            </div>
+        </div>
+        <div class="form-group row">
+            <label for="headers" class="col-sm-3 control-label text-right" translate>Headers (JSON format)</label>
+            <div class="col-sm-9">
+                <textarea name="headers" id="headers" ng-model="params.headers"
+                    class="form-control" required>{{ params.headers }}</textarea>
+            </div>
+        </div>
+        <div class="form-group row">
+            <label for="responsemapping" class="col-sm-3 control-label text-right" translate>Response Mapping (JSON format)</label>
+            <div class="col-sm-9">
+                <input placeholder="{'username': '{your.response.user.name.key'}" name="responsemapping" id="responsemapping"
+                    ng-model="params.responseMapping" class="form-control" required />
+            </div>
+        </div>
+        <div class="form-group row">
+            <label for="hasSpecialErrorHandler" class="col-sm-3 control-label text-right" translate>Special Error
+                Handling</label>
+            <div class="col-sm-3">
+                <input name="hasSpecialErrorHandler" id="hasSpecialErrorHandler" type="checkbox"
+                    ng-model="params.hasSpecialErrorHandler" ng-true-value="'True' || true" ng-false-value="'False' || false" />
+            </div>
+        </div>
+        <div class="form-group row" ng-if="params.hasSpecialErrorHandler">
+            <label for="errorResponseMapping" class="col-sm-3 control-label text-right" translate>Response contains (JSON format):</label>
+            <div class="col-sm-9">
+                <textarea name="errorResponseMapping" id="errorResponseMapping" ng-model="params.errorResponseMapping" class="form-control"></textarea>
+            </div>
+        </div>
+        <div class="text-center" ng-show="showResult">
+            <label class="label" ng-class="{'label-success':result.result,
+                       'label-danger':!result.result}">
+                {{ result.description }}<span class="glyphicon glyphicon-remove" ng-click="showResult=false"></span>
+            </label>
+        </div>
+        <div class="container-fluid">
+            <div class="text-right">
+                <div class="form-group row form-inline">
+                    <input name="testEmail" id="testEmail" class="form-control" ng-model="params.testEmail"
```

Comment:
i think we should not use *email* here, we query the *userid* (whether it is an email or something else).